nx-X11/include/extensions/Imakefile

```diff
@@ -47,9 +47,6 @@ RENDERHEADERS = render.h renderproto.h
 #if BuildXfixes || BuildXfixesLibrary
 XFIXESHEADERS = xfixeswire.h xfixesproto.h
 #endif
-#if BuildDamage || BuildDamageLibrary
-DAMAGEHEADERS = damagewire.h damageproto.h
```

Comment:
Both files can probably be removed from `debian/copyright.in` (although I remember we've been talking about that... but not the outcome of this conversation. Leaving it in copyright.in is probably fine anyway.)